src/main.ts

```diff
@@ -148,8 +146,18 @@ async function getAIResponse(prompt: string): Promise<Array<{
       ],
     });
 
-    const res = response.choices[0].message?.content?.trim() || "{}";
-    return JSON.parse(res).reviews;
+    const rawContent = response.choices[0].message?.content || "";
+
+    const startIndex = rawContent.indexOf("{");
+    const endIndex = rawContent.lastIndexOf("}") + 1;
+
+    if (startIndex === -1 || endIndex === -1) {
+      throw new Error("No valid JSON found in response content.");
+    }
+
+    const jsonString = rawContent.substring(startIndex, endIndex).trim();
```

Comment:
When there was no `json_object` a long time ago, my solution to this problem was to find the first "{" and the last "}" in the string, and then we parsed it